docs/plugin-development.rst

```diff
@@ -30,7 +30,7 @@ our example, we'll just use JSON. ::
     touch cats/plugin.json
 
 The plugin config file should specify a human-readable name and description for your
-plugin, and can optionally contain a list of other plugins that your plugin
+plugin, and can optionally contains a URL to a documentation, a list of other plugins that your plugin
```

Comment:
@msmolens Thank you for the suggestion. I'll update with the suggested version. 👍